test/python/install_python311_setuptools_vulnerability.sh

```diff
@@ -79,8 +79,23 @@ checkVulnerableDir()
     fi
 }
 
+checkVulnerableFile()
+{
+    FILE=$1
+    VERSION=$2
+    
+    if [[ -f $FILE ]] ; then
+        echoStderr "❌ check for vulnerable setuptools version failed for python ${VERSION}."
+        return 1
+    else
+        echo "✅ Passed! Either the container does not have vulnerable version or vulnerable version specific file got removed."
+        return 0
+    fi
+}
+
 # only for 3.11
```

Comment:
I remember you added a `find` command which lists all the files containing `setuptools`. However, I don't see it anymore. 

Can we add that back?

Reply:
added it back. thanks for reminder.